src/drivers/spi/dw_spi/Mybuild

```diff
@@ -0,0 +1,17 @@
+package embox.driver.spi
+
+module dw_spi {
+	option number base_addr0 = 0
```

Comment:
In my opinion, it's better to extract common code which initialize (jr create) controller, and make 4 different module dw_spi_1 dw_spi_2 and so on